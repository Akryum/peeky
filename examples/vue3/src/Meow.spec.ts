// import { describe, test, expect } from '@peeky/test'
import { mount } from '@vue/test-utils'
import Meow from './Meow.vue'

test('meow', async () => {
  const wrapper = mount(Meow)

  expect(wrapper.html()).toMatchSnapshot()
})
