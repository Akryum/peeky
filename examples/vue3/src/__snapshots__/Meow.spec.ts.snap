// Peeky snapshots v1.0

exports[`Meow meow 1`] = `
<button
  class="meow"
  data-v-1e4fbaf9
>
  Meow
</button>
`;